### PR TITLE
Add ban message for Minecraft: Java Edition

### LINKED ISSUE #191: Adding a banned message to the MC project
With 1.19.1 adding account bans soon we should probably add the banned helper message from the MCPE project to the MC project. It might also need some touchups like the Case Review Request Link: https://help.minecraft.net/hc/en-us/requests/new?ticket_form_id=360003469452

### PR DESCRIPTION
**Pull request by dericksonmark**
Closes #191, also fixes a broken link. Draft, as I need to ensure the correct link is being used.
Feedback welcome :)

### CODE REVIEW

**Comment by Marcono1234** [comment]
Thanks for creating this message!

Maybe for Java Edition we should have a separate message which links to https://aka.ms/chatreportingfaq.
I am also not sure if we should directly refer to the appeal form, or whether the players should read the FAQs and find the link to it there. That would (hopefully) make sure that they understand better why they might have been banned. What do you think?

File: assets/messages.yml [hunk 1]
```diff
@@ -216,7 +217,8 @@ messages:
         However, this issue is {color:#FF5722}*Invalid*{color}.
 
         This is an account issue. We do not have the tools to help you with this issue on this bug tracker.
-        Please have a look at the help article "[Why have I been banned from Minecraft?|https://help.minecraft.net/hc/en-us/articles/360052618531-Why-have-I-been-banned-from-Minecraft-]" and the [Community Standards|https://www.minecraft.net/en-us/community-standards].
+        Please have a look at the help article "[Why have I been banned from Minecraft?|https://help.minecraft.net/hc/en-us/articles/4408964729869-Why-Have-I-Been-Banned-from-Minecraft-]" and the [Community Standards|https://www.minecraft.net/en-us/community-standards].
+        To request a case review of a ban, please contact Mojang Support through [this page|https://help.minecraft.net/hc/en-us/requests/new?ticket_form_id=360003469452].
```
Comment (osfanbuff63) [inline assets/messages.yml:221]: Either way, this should be added imo to the normal one, unless it's only for Java

**Comment by dericksonmark** [comment]
> Maybe for Java Edition we should have a separate message which links to https://aka.ms/chatreportingfaq. I am also not sure if we should directly refer to the appeal form, or whether the players should read the FAQs and find the link to it there. That would (hopefully) make sure that they understand better why they might have been banned. What do you think?

Ooh, that sounds like a good idea. :) I'll add it.

File: assets/messages.yml [hunk 2]
```diff
@@ -216,7 +216,22 @@ messages:
...
+    - project:
+        - mc
+      name: Banned
+      shortcut: ban
+      message: |-
+        *Thank you for your report!*
+        However, this issue is {color:#FF5722}*Invalid*{color}.
+        
+        This is an account issue. We do not have the tools to help you with this issue on this bug tracker.
+        Please have a look at the help article "[Why have I been banned from Minecraft?|https://help.minecraft.net/hc/en-us/articles/4408964729869-Why-Have-I-Been-Banned-from-Minecraft-]" and the [Community Standards|https://www.minecraft.net/en-us/community-standards].
```
Comment (Marcono1234) [inline assets/messages.yml:233]: "Why Have I Been Banned from Minecraft?" seems to be mostly about Bedrock edition at the moment, the screenshot is from Bedrock edition and the text says:
> when they attempt to sign into Minecraft on any platform (non-Java Edition)

So maybe we should omit that link for now?

Comment (dericksonmark) [reply assets/messages.yml:233]: Ah, I missed that. I'll change it. Thanks :)

**Comment by Marcono1234** [comment]
Do you want to mark this pull request as "Ready for review", or you do want to wait until chat reporting is fully released, in case further information is published by Mojang?

**Comment by dericksonmark** [comment]
It's a draft mostly because I'm waiting to see if there's an aka.ms link for the Case Review link, so that it can more dynamically update if Mojang decides to switch the links they use on the support page.

**Review by osfanbuff63 — APPROVED** [review]

**Review by Marcono1234 — APPROVED** [review]

**Pull request merged**